partials/message.html

```diff
@@ -2,6 +2,14 @@
 	<div class="o-message__container">
 		<div class="o-message__content">
 			<p class="o-message__content-main">{{message}}</p>
+
+			{{#if actions}}
+				<div class="o-message__actions">
+					{{#each actions}}
+						<a href="{{link}}" class="o-message__actions__{{#if isSecondary}}secondary{{else}}primary{{/if}}">{{text}}</a>
```

Comment:
I would not break up the classes, this makes it easier to find if we need to change the classes ever.

```suggestion
						<a href="{{link}}" class="{{#if isSecondary}}o-message__actions__secondary{{else}}o-message__actions__primary{{/if}}">{{text}}</a>
```